Graphs/AbstractGraph.js

```diff
@@ -697,4 +697,27 @@ export default class AbstractGraph extends React.Component {
             );
         })
     }
+
+    // Get the string by available width (in px) and if required, then will wrap elipsis "..." as well
+    wrapTextByWidth(text, width) {
```

Comment:
Any reason we cannot change the `wrapD3Text` function as the overlapping will apply to all graphs?

Reply:
@natabal Actually, in the case of Vertical Bar Graph, we cannot use the width of a bar, so we need to restrict the label with a certain number of characters.
But in case of Horizontal Bar, we already had Bar Width, and need to adjust the characters within the width, not by the no. of characters.